src/components/ContactPage/ContactSection/index.tsx

```diff
@@ -16,8 +16,13 @@ const ContactSection: React.FC = () => {
             photography, we're here to bring your vision to life.
           </p>
         </div>
-        <div className="md:w-auto md:ml-4 flex justify-center md:justify-end">
-          <SocialMediaButtons />
+        <div
+          className="w-1/2 h-80 rounded-3xl overflow-hidden bg-cover bg-center relative"
+          style={{ backgroundImage: "url('/kytsya.jpg')" }}
+        >
+          <div className="bottom-1/2 -right-4 bg-main-color px-4 py-4 rounded-full absolute">
+            <SocialMediaButtons className="rounded-full border border-gray-600 px-2 py-2" />
+          </div>
```

Comment:
I'm  not sure you need absolute here, let's discuss

Reply:
with
![image](https://github.com/user-attachments/assets/6b8fbe50-3e3b-4398-8d54-717d9b5e6b47)

Reply:
without 
![image](https://github.com/user-attachments/assets/0fbed74e-c4ac-4d83-98ad-8fbfdae508af)